lib/msf/core/payload_generator.rb

```diff
@@ -366,6 +366,10 @@ def generate_java_payload
     # methods in order based on the supplied options and returns the finished payload.
     # @return [String] A string containing the bytes of the payload in the format selected
     def generate_payload
+      if payload.include?("pingback") and framework.db.active == false
```

Comment:
Minor: Per The Ruby Style Guide[No and or or](https://github.com/rubocop-hq/ruby-style-guide#no-and-or-or)

Reply:
Meh; that matches the coding style used throughout that file.